public/app/plugins/datasource/graphite/configuration/ConfigEditor.tsx

```diff
@@ -13,6 +14,7 @@ const graphiteVersions = [
   { label: '1.0.x', value: '1.0' },
   { label: '1.1.x', value: '1.1' },
 ];
+const DEFAULT_GRAPHITE_VERSION = graphiteVersions[2];
```

Comment:
Nitpick: probably makes sense to have `last(graphiteVersions)` so that when new version is added it's automatically the latest version

Reply:
I've extracted the list to `graphite_versions.ts` to use the default (now last from the list) in `datasource.ts` and `ConfigEditor.tsx`